nameserver/server.go

```diff
@@ -72,17 +146,16 @@ func rdnsHandler(config *dns.ClientConfig, lookups []Lookup) dns.HandlerFunc {
 	}
 }
 
-/* When we receive a request for a name outside of our '.weave.local.'
-   domain, ask the configured DNS server as a fallback.
-*/
-func notUsHandler(config *dns.ClientConfig) dns.HandlerFunc {
+// When we receive a request for a name outside of our '.weave.local.'
+// domain, ask the configured DNS server as a fallback.
+func (s *DNSServer) notUsHandler() dns.HandlerFunc {
```

Comment:
I did it for coherence with the creation of the `DNSServer` class: once you have the class, all the initialization parameters should be provided in the `New*()` methods, and that means that you store the config as an attribute in the instance... then it didn't make sense to pass it around to handlers anymore...